.github/workflows/release.yml

```diff
@@ -5,9 +5,6 @@ on:
     types:
       - published
 
-env:
-  PYTHONUNBUFFERED: 1
-
```

Comment:
This change is unrelated to BUILD-2320, next time will be better to do it via separate PR.